dashboard/app/controllers/pd/regional_partner_mini_contact_controller.rb

```diff
@@ -1,6 +1,4 @@
 class Pd::RegionalPartnerMiniContactController < ApplicationController
-  load_resource :regional_partner_mini_contact, class: 'Pd::RegionalPartnerMiniContact', id_param: :contact_id, only: [:thanks]
```

Comment:
Why are we removing this line?

Reply:
Probably should be in a separate PR, but as far as I can tell there is no thanks action for the mini contact controller (think this was just copied from the full contact controller). It appears [thanks is just handled in the React component.](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/pd/regional_partner_mini_contact/RegionalPartnerMiniContact.jsx#L143)

Reply:
Ah, understood.  All good then!

Reply:
There was a `thanks` route that I hadn't removed -- just removed in a new commit, and checked via google analytics that in the past 2 months a regional_partner_mini_contact/thanks URL has never been reported.